app/_src/production/dp-config/dpp-on-kubernetes.md

```diff
@@ -180,6 +180,18 @@ On Kubernetes, `Dataplane` resource is automatically created by kuma-cp. For eac
 
 To join the mesh in a graceful way, we need to first make sure the application is ready to serve traffic before it can be considered a valid traffic destination.
 
+{% if_version lte:2.3.x %}
+
+If any of the init containers experience network connectivity issues we suggest trying setting `runtime.kubernetes.injector.sidecarContainer.waitForDataplaneReady` to `true` so that the init container for the sidecar finishes only when the sidecar is ready to serve traffic.
+
+{% warning %}
+The `waitForDataplaneReady` setting relies on Kubernetes container creation sequence (which is undocumented, so it might be changed in the future) and injecting kuma-init as the first init container (which we can't guarantee, because other mutating webhooks can rearrange the init containers) with a `postStart` hook.
```

Comment:
I'd rewrite this warning like this:

```
The waitForDataplaneReady setting relies on the way that Kubernetes creates containers, which is not documented and could change in the future. It also relies on the Kuma init container being injected as the first init container in the pod, which is not guaranteed because other mutating webhooks can rearrange the init containers.

A better solution will be available when sidecar containers are more stable and widely available.
```

Parentheses are usually better to be avoided.